EzMap.Api/Controllers/PoiController.cs

```diff
@@ -71,4 +71,32 @@ public async Task<IActionResult> Delete(Guid id, [FromServices] IUnitOfWork uow)
 
         return new StatusCodeResult(StatusCodes.Status500InternalServerError);
     }
+
+    [Authorize]
+    [HttpGet("{id:guid}")]
+    public async Task<IActionResult> GetPoiDetails(Guid id, [FromServices] IUnitOfWork uow)
+    {
+        if (string.IsNullOrEmpty(id.ToString()))
+        {
+            return BadRequest("Please provide a valid id!");
+        }
+
+        var result = await uow.PoiRepository.GetPoiById(id);
+
+        if (result is not null)
+        {
+            return Ok(result);
+        }
+        
+        return new StatusCodeResult(StatusCodes.Status500InternalServerError);
+    }
+
+    [Authorize]
+    [HttpGet]
+    public async Task<IActionResult> GetListPoi([FromServices] IUnitOfWork uow)
```

Comment:
So this means we get all Pois regardless of who owns/created the Poi right?
Better get the domain logic right before implementing.